app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,24 @@
  *= require font-awesome
  */
 
+
+.searchLocation {
```

Comment:
Rule set contains (14/10) properties<br>Selector `searchLocation` should be written in lowercase with hyphens